docs/docs-content/clusters/data-center/vmware.md

```diff
@@ -1072,6 +1049,38 @@ In addition to the default cloud account already associated with the private clo
 | **Username** | vCenter username|
 | **Password** | vCenter password|
 
+:::caution
+
+If you change the password for a user account in vCenter, you must also change it in Palette for the same VMware cloud account. We recommend updating the passwords immediately to avoid potentially locking Palette out of vCenter. For guidance, refer to [Change VMware Cloud Account Password in Palette](/clusters/data-center/vmware#change-vmware-cloud-account-password-in-palette).
+
+:::
+
+
+## Change VMware Cloud Account Password in Palette
```

Comment:
```suggestion
## Change VMware Cloud Account Password
```
I'm trying to think of ways to shorten the title. So far the best idea I have is to remove "in Palette". Thoughts?

Reply:
@karl-cardenas-coding I also pondered how to shorten it. I'll delete "in Palette".